react/ContextHeader/styles.styl

```diff
@@ -0,0 +1,31 @@
+@require 'settings/palette.styl'
+
+.context-header
+  margin 1rem 2rem
+  background paleGrey
+  border 1px solid silver
+  padding 1rem
+  border-radius .3rem
+  display flex
+  flex-grow 1
+  flex-shrink 0
+  flex-direction row
+  justify-content flex-start
+  align-items center
+
+.context-header--content
```

Comment:
Not sure you're using BEM quite right. It's kinda `block-element--modifier` and `content`, `title`, `text` & `icon` are really element, not modifiers.
I would go for something like this `context-header-content` or simply `contextheader-content` which is closer to BEM style but not as easily readable as the first one.